recipes/runit_systemd.rb

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright:: Copyright (c) 2015 Opscode, Inc.
+#
+# All Rights Reserved
+#
+
+project_name = node['enterprise']['name'].clone
+project_name.gsub!(/_/, '-') if project_name == 'private_chef'
+unit_name = "#{project_name}-runsvdir-start.service"
```

Comment:
i thought we only had enterprise chef now?  and this is a bit convoluted and the clone can be dropped if gsub! is not used.